Sources/Operators.swift

```diff
@@ -16,6 +16,10 @@ import Foundation
 
 // MARK: - <| Operator
 
+precedencegroup Decodative {
+    associativity: left
```

Comment:
It is but I don't know if it makes much of a difference. I wasn't able to determine if the default group has associativity defined. If it does then we can just leave it as default and I'll remove these lines.